packages/vitest-plugin-storybook/src/plugin.ts

```diff
@@ -38,6 +40,10 @@ export const storybookTest = (options?: UserOptions): any => {
 
   const storybookUrl = finalOptions.storybookUrl || defaultOptions.storybookUrl
 
+  // To be accessed by the global setup file
+  process.env.__STORYBOOK_URL__ = storybookUrl
+  process.env.__STORYBOOK_SCRIPT__ = finalOptions.storybookScript
```

Comment:
will this make it also available for the browser?

Reply:
no